scripts/apps/authoring-react/authoring-integration-wrapper.tsx

```diff
@@ -447,14 +451,36 @@ export class AuthoringIntegrationWrapper extends React.PureComponent<IPropsWrapp
                             authoringStorage,
                             fieldsAdapter,
                             storageAdapter,
+                            spellchecker,
                         }) => {
                             const authoringActionsFromExtensions = getAuthoringActionsFromExtensions(
                                 item,
                                 contentProfile,
                                 fieldsData,
                             );
 
+                            const getSpellcheckerAction = (): IAuthoringAction => {
+                                if (appConfig.features.useTansaProofing !== true) {
+                                    return {
+                                        label: spellchecker.enabled
+                                            ? gettext('Disable spellchecker')
+                                            : gettext('Enable spellchecker'),
+                                        onTrigger: () => {
+                                            spellchecker.toggleSpellchecker(!spellchecker.enabled);
+                                        },
+                                        keyBindings: {
+                                            'ctrl+shift+y': () => {
+                                                spellchecker.toggleSpellchecker(!spellchecker.enabled);
+                                            },
+                                        },
+                                    } as IAuthoringAction;
+                                }
+
+                                return {} as IAuthoringAction;
```

Comment:
If you don't have an item just don't put it into the array instead of putting a fake item and casting to make the error go away